model/reviews-model.js

```diff
@@ -2,7 +2,15 @@ const db = require("../db/connection");
 
 exports.fetchReviewById = (id) => {
   return db
-    .query("SELECT * FROM reviews WHERE review_id = $1", [id])
+    .query(
+      `SELECT reviews.*,
+       CAST(COUNT(comments.comment_id) AS INTEGER) AS comment_count 
+       FROM reviews JOIN comments 

```

Comment:
What happens if there is a review with no comments? What kind of JOIN should this be? Have a look at the sql relationships notes for this.